docs/guides/mssqlserver/backup/auto-backup/index.md

```diff
@@ -0,0 +1,903 @@
+
+Now, we have to create a `BackupBlueprint` CR with a blueprint for `BackupConfiguration` object.
+
+```yaml
+apiVersion: core.kubestash.com/v1alpha1
+kind: BackupBlueprint
+metadata:
+  name: mssqlserver-customize-backup-blueprint
+  namespace: demo
+spec:
+  usagePolicy:
+    allowedNamespaces:
+      from: All
+  backupConfigurationTemplate:
+    deletionPolicy: OnDelete
+    backends:
+      - name: gcs-backend
+        storageRef:
+          namespace: demo
+          name: gcs-storage
+        retentionPolicy:
+          name: demo-retention
+          namespace: demo
+    sessions:
+      - name: frequent-backup
+        sessionHistoryLimit: 3
+        scheduler:
+          schedule: ${schedule}
+          jobTemplate:
+            backoffLimit: 1
+        repositories:
+          - name: ${repoName}
+            backend: gcs-backend
+            directory: ${namespace}/${targetName}
+            encryptionSecret:
+              name: encrypt-secret
+              namespace: demo
+        addon:
+          name: mssqlserver-addon
+          jobTemplate:
+            spec:
+              securityContext:
+                runAsUser: 0
+          tasks:
+            - name: logical-backup
+              params:
+                databases: ${targetedDatabase}
+```
+
+Note that we have used some variables (format: `${<variable name>}`) in different fields. KubeStash will substitute these variables with values from the respective target’s annotations. You’re free to use any variables you like.
+
+Here,
+
+- `.spec.backupConfigurationTemplate.backends[*].storageRef` refers our earlier created `gcs-storage` backupStorage.
+- `.spec.backupConfigurationTemplate.sessions[*]`:
+    - `.schedule` defines `${schedule}` variable, which determines the time interval for the backup.
+    - `.repositories[*].name` defines the `${repoName}` variable, which specifies the name of the backup `Repository`.
+    - `.repositories[*].directory` defines two variables, `${namespace}` and `${targetName}`, which are used to determine the path where the backup will be stored.
+    - `.addon.tasks[*].params.args` defines `${targetedDatabase}` variable, which identifies list of databases to backup.
+
+Let's create the `BackupBlueprint` we have shown above,
+
+```bash
+$ kubectl apply -f https://github.com/kubedb/docs/raw/{{< param "info.version" >}}/docs/guides/mssqlserver/backup/auto-backup/examples/customize-backupblueprint.yaml
+backupblueprint.core.kubestash.com/mssqlserver-customize-backup-blueprint created
+```
+
+Now, we are ready to backup our `MSSQLServer` databases using few annotations. You can check available auto-backup annotations for a databases from [here](https://kubestash.com/docs/latest/concepts/crds/backupblueprint/).
+
+**Create Database**
+
+Now, we are going to create an `MSSQLServer` CR in demo namespace. Below is the YAML of the `MSSQLServer` object that we are going to create,
```

Comment:
We will now deploy an SQL Server Availability Group cluster by creating an MSSQLServer CR in the demo namespace. Below is the YAML configuration for the MSSQLServer object we are about to create: